portfolio.html

```diff
@@ -6,21 +6,25 @@
 		<title>My Portfolio</title>
```

Comment:
I didn't see a DOCTYPE but that might just be github.

Reply:
I think it was github, because it is in the original doc. Thanks.